mantidproject/mantid
# Remove 'None' title in the legend of 1D plots

**Description of work.**
This issue was fixed in a previous PR (#28147), but in order to allow for unit testing I have moved the location of this fix to a different function.

**To test:**

1. Open workbench on Ubuntu ( but presumably other distributions (and possibly mac?)
2. Plot any data
3. Right-click and add errors
4. A spurious 'None' entry should no longer appear.



*Associated issue has been fixed in a previous PR.*




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
